### PR TITLE
Counter down - redirection to map view

### PR DESCRIPTION
**Pull request by luis-tm**
- Included countdown as a component.
- Included countdown in lobby view: if all players are ready, a timer starts and when the time is off, the host player calls create-game. The server then calls back all the players in the client, who are redirected to the game view. The players can abort the timer by clicking "not ready".
- Modified socket to send received message to onMessage function.
- Fixed problems in QR Scan view.

### CODE REVIEW

**Comment by luis-tm** [comment]
image

**Comment by luis-tm** [comment]
image

**Review by paolovic — APPROVED** [review]
great! ty

File: src/components/ui/Countdown.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { CountdownCircleTimer } from "react-countdown-circle-timer";
+import { Button } from 'components/ui/Button';
+import "styles/ui/Countdown.scss";
+
+const Countdown = props => {
+
+    return (
+        <div className="countdown-container">
```
Comment (apple00juice) [inline src/components/ui/Countdown.js:8]: Do we want to add the backdrop like the CustomPopUp, then you can also control if it is displayed with the open property of the backdrop?

**Review by apple00juice — APPROVED** [review]
Looks really good 👍

**Review by paolovic — APPROVED** [review]

File: src/components/ui/Countdown.js (hunk 1 above)
Comment (luis-tm) [reply src/components/ui/Countdown.js:8]: I tried to implement it ... but although the countdown is not shown when open={false}, it's actually working, and when the time goes to 0, the countdown action is triggered :(  I'll see if I can do smth to add it.

Comment (luis-tm) [reply src/components/ui/Countdown.js:8]: ah great! and we still can use the other properties of the brackdrop 👍

**Pull request merged**